src/meetup.py

```diff
@@ -0,0 +1,86 @@
+# this class is about to collect all meetups from einundzwanzig.space/meetups.json
+# and returns a (Telegram-)message with all meetups (command: /meetup)
+
+import config
+
+from telegram.ext.callbackcontext import CallbackContext
+from telegram.update import Update
+
+import requests
+
+class Meetup:
+
+    def __init__(self, name: str, region: str, url: str) -> None:
+        self.name = name
+        self.region = region
+        self.url = url
+
+
+def show_meetups(update: Update, context: CallbackContext):
+
+
+    # get json file from website
+    url = requests.get(f'{config.EINUNDZWANZIG_URL}/meetups.json', timeout=5)
+    json_string = url.text
+
+    # save json file as string
+    json_string = str(json_string)

```

Comment:
This doesn't really make sense as I explained during the last review. Just use your JSON object, not the string version. Have a look at the soundboard code to see how to accomplish this. It doesn't make sense to count characters in the json string if the json is already correctly parsed, meaning you just need to go through the array.